src/sys/event.rs

```diff
@@ -86,131 +169,194 @@ libc_enum! {
                   target_os = "freebsd",
                   target_os = "dragonfly"))]
+        #[allow(missing_docs)]
         NOTE_FFCTRLMASK;
         #[cfg(any(target_os = "macos",
                   target_os = "ios",
                   target_os = "freebsd",
                   target_os = "dragonfly"))]
+        #[allow(missing_docs)]
         NOTE_FFLAGSMASK;
         #[cfg(any(target_os = "macos",
                   target_os = "ios",
                   target_os = "freebsd",
                   target_os = "dragonfly"))]
+        #[allow(missing_docs)]
         NOTE_FFNOP;
         #[cfg(any(target_os = "macos",
                   target_os = "ios",
                   target_os = "freebsd",
                   target_os = "dragonfly"))]
+        #[allow(missing_docs)]
         NOTE_FFOR;
+        #[allow(missing_docs)]
         NOTE_FORK;
+        #[allow(missing_docs)]
         NOTE_LINK;
+        #[allow(missing_docs)]
         NOTE_LOWAT;
         #[cfg(target_os = "freebsd")]
+        #[allow(missing_docs)]
         NOTE_MSECONDS;
         #[cfg(any(target_os = "macos", target_os = "ios"))]
+        #[allow(missing_docs)]
         NOTE_NONE;
         #[cfg(any(target_os = "macos", target_os = "ios", target_os = "freebsd"))]
+        #[allow(missing_docs)]
         NOTE_NSECONDS;
         #[cfg(target_os = "dragonfly")]
+        #[allow(missing_docs)]
         NOTE_OOB;
+        #[allow(missing_docs)]
         NOTE_PCTRLMASK;
+        #[allow(missing_docs)]
         NOTE_PDATAMASK;
+        #[allow(missing_docs)]
         NOTE_RENAME;
+        #[allow(missing_docs)]
         NOTE_REVOKE;
         #[cfg(any(target_os = "macos", target_os = "ios", target_os = "freebsd"))]
+        #[allow(missing_docs)]
         NOTE_SECONDS;
         #[cfg(any(target_os = "macos", target_os = "ios"))]
+        #[allow(missing_docs)]
         NOTE_SIGNAL;
+        #[allow(missing_docs)]
         NOTE_TRACK;
+        #[allow(missing_docs)]
         NOTE_TRACKERR;
         #[cfg(any(target_os = "macos",
                   target_os = "ios",
                   target_os = "freebsd",
                   target_os = "dragonfly"))]
+        #[allow(missing_docs)]
         NOTE_TRIGGER;
         #[cfg(target_os = "openbsd")]
+        #[allow(missing_docs)]
         NOTE_TRUNCATE;
         #[cfg(any(target_os = "macos", target_os = "ios", target_os = "freebsd"))]
+        #[allow(missing_docs)]
         NOTE_USECONDS;
         #[cfg(any(target_os = "macos", target_os = "ios"))]
+        #[allow(missing_docs)]
         NOTE_VM_ERROR;
         #[cfg(any(target_os = "macos", target_os = "ios"))]
+        #[allow(missing_docs)]
         NOTE_VM_PRESSURE;
         #[cfg(any(target_os = "macos", target_os = "ios"))]
+        #[allow(missing_docs)]
         NOTE_VM_PRESSURE_SUDDEN_TERMINATE;
         #[cfg(any(target_os = "macos", target_os = "ios"))]
+        #[allow(missing_docs)]
         NOTE_VM_PRESSURE_TERMINATE;
+        #[allow(missing_docs)]
```

Comment:
If we find ourselves sprinkling `#[allow(missing_docs)]` over every attribute like this in other structs, we might consider modifying  the `libc_bitflags` macro to make it less tedious.